### ISSUE TITLE
Unexpected error if result.warning() node is Root

### ISSUE THREAD

**Issue by davidtheclark** [postcss/postcss#526]
If the `node` option of `result.warning()` is the `Root` node, I hit this error:

```
TypeError: Cannot read property 'line' of undefined
    at new Warning (/Users/davidclark/dev/stylelint/node_modules/postcss/lib/warning.js:20:28)
    at Result.warn (/Users/davidclark/dev/stylelint/node_modules/postcss/lib/result.js:40:28)
```

**Comment by ai** [comment]
Fixed 35045a9

**Comment by ai** [comment]
Released in 5.0.3.
